### PR TITLE
Dp 21764 data listing views with styles

### PR DESCRIPTION
**Pull request by clairesunstudio**
https://massgovfeature1.prod.acquia-sites.com/mockup-massachusetts-data-hub---unpublished?auHash=83aHo2s6ddHVjwE13yBwZN47Mrjw7NATBWQWQtGukcs

### CODE REVIEW

**Comment by clairesunstudio** [comment]
deploy to feature1 https://massgovfeature1.prod.acquia-sites.com/data-listing/all

**Comment by clairesunstudio** [comment]
deploy to feature1 https://massgovfeature1.prod.acquia-sites.com/data-listing/all

File: docroot/themes/custom/mass_theme/templates/views/views-view--data-listing.html.twig [hunk 1]
```diff
@@ -32,40 +33,55 @@
...
   set classes = [
     dom_id ? 'js-view-dom-id-' ~ dom_id,
+    'main-content',
+    'main-content--two main-content--reversed',
+    'data-listing-wrapper'
   ]
 %}
 <div{{ attributes.addClass(classes) }}>
+  <div class="ma__breadcrumb--views">{{ header.area }}</div>
   {{ title_prefix }}
```
Comment (clairesunstudio) [inline docroot/themes/custom/mass_theme/templates/views/views-view--data-listing.html.twig]:
```suggestion
```

File: docroot/themes/custom/mass_theme/templates/views/views-view--data-listing.html.twig [hunk 2]
```diff
@@ -32,40 +33,55 @@
...
-  {{ title }}
+  <h1>{{ title }}</h1>
   {{ title_suffix }}
```
Comment (clairesunstudio) [inline docroot/themes/custom/mass_theme/templates/views/views-view--data-listing.html.twig]:
```suggestion
```

File: conf/drupal/config/views.view.data_listing_all.yml [hunk 3]
```diff
@@ -81,13 +81,13 @@ display:
       pager:
         type: full
         options:
-          items_per_page: 10
+          items_per_page: 5
```
Comment (weekbeforenext) [inline conf/drupal/config/views.view.data_listing_all.yml]: Do we want this set to 10?

File: conf/drupal/config/views.view.data_listing_topic.yml [hunk 4]
```diff
@@ -372,7 +372,7 @@ display:
           empty: false
           tokenize: false
           content:
-            value: '<p><a href="/data-listing-all">See All Data Listing »</a></p>'
+            value: '<div class="ma__breadcrumb--views"><ol><li><a href="https://edit.mass.gov/mockup-massachusetts-data-hub---unpublished?auHash=83aHo2s6ddHVjwE13yBwZN47Mrjw7NATBWQWQtGukcs">Data Home</a></li><li><a href="/data-listing/all">All Data Listing</a></li></ol></div>'
```
Comment (weekbeforenext) [inline conf/drupal/config/views.view.data_listing_topic.yml:375]: Same as other URL to unpublished page.

File: conf/drupal/config/views.view.data_listing_all.yml [hunk 5]
```diff
@@ -372,6 +372,19 @@ display:
...
+        area:
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: true
+          tokenize: false
+          content:
```
Comment (weekbeforenext) [inline conf/drupal/config/views.view.data_listing_all.yml:384]: This URL will change when the page is published. Should we set an alias in Prod and use that or just note that this should be updated before merge/deploy?

**Review by weekbeforenext — COMMENTED** [review]
Looks good with a couple of tweaks.

File: conf/drupal/config/views.view.data_listing_all.yml (hunk 5 above)
Comment (clairesunstudio) [reply conf/drupal/config/views.view.data_listing_all.yml:384]: I'm ok either way. Where can I set the aliases?

File: conf/drupal/config/views.view.data_listing_all.yml (hunk 3 above)
Comment (clairesunstudio) [reply conf/drupal/config/views.view.data_listing_all.yml]: yep I will revert this change

**Comment by clairesunstudio** [comment]
Fix major IE11 filter toggle functionality issue
Screen Shot 2021-05-25 at 1 58 06 PM

We are ignoring the minor visual degradation on IE11 in the teaser eyebrow
Screen Shot 2021-05-25 at 1 35 47 PM

**Comment by clairesunstudio** [comment]
Released to feature1

**Pull request merged**